static/app/views/settings/components/dataScrubbing/index.tsx

```diff
@@ -211,10 +211,9 @@ export function DataScrubbing({
             disabled={disabled}
           />
         ) : (
-          <EmptyMessage
-            icon={<IconWarning size="xl" />}
-            description={t('You have no data scrubbing rules')}
-          />
+          <EmptyMessage icon={<IconWarning size="xl" />}>
+            {t('You have no data scrubbing rules')}
+          </EmptyMessage>
```

Comment:
Slots might be a perfect use-case for the empty state component, that way everything could be a child, and we'd take care of the placement internally